src/System.Text.Utf8/src/System/Text/Utf16/Utf16LittleEndianEncoder.cs

```diff
@@ -7,7 +7,7 @@ public static class Utf16LittleEndianEncoder
     {
         const uint MaskLow10Bits = 0x3FF;
 
-        public static bool TryDecodeCodePoint(ByteSpan buffer, out UnicodeCodePoint codePoint, out int encodedBytes)
+        public static bool TryDecodeCodePoint(Span<byte> buffer, out UnicodeCodePoint codePoint, out int encodedBytes)
```

Comment:
My only question is why we have a Span<T> in System.Buffers and a Span<T> in System.Buffers. What is the difference between which one and I am imagining we will remove one of them. Is that correct?